pkg/envoy/ads/stream.go

```diff
@@ -20,6 +22,42 @@ import (
 	"github.com/openservicemesh/osm/pkg/utils"
 )
 
+// This is the workqueue Job implementation to issue a Send Response to a proxy
+type sendJob struct {
```

Comment:
Would it make sense to move this to `pkg/envoy/ads/types.go`?

Reply:
we could. Since issuing this type of job is currently used only in this file, I thought here would be fine but I don't mind the change